submission/submission-webapp/src/itest/kotlin/ac/uk/ebi/biostd/itest/test/project/submit/ExtCollectionSubmitTest.kt

```diff
@@ -13,8 +13,8 @@ import ac.uk.ebi.biostd.persistence.repositories.SequenceDataRepository
 import ebi.ac.uk.asserts.assertThat
 import ebi.ac.uk.dsl.line
 import ebi.ac.uk.dsl.tsv
+import ebi.ac.uk.extended.model.ExtCollection
```

Comment:
We'll need to change the package name here `ac.uk.ebi.biostd.itest.test.collection.submit`